### DISCUSSION TITLE
PHP Functionality

### DISCUSSION THREAD

**Discussion by ECLAYA4188** [jlesage/docker-nginx-proxy-manager#298] [Q&A]
I am having trouble figuring out how to use this docker with some website that use php files.  Particularly my pihole servers that are located on other raspberry pi on my LAN. They are running 64bit Debian 11. I am running another pho docker on same server as this nginx docker. Any insight on how to properly configure?

**Comment by jlesage** [comment]
I'm not sure I understand the question.  Are you trying to host a website using Nginx Proxy Manager ?  Note that Nginx Proxy Manager acts as a proxy and is not a web server.
